src/main/java/edu/kis/powp/jobs2d/command/Visitable.java

```diff
@@ -0,0 +1,8 @@
+package edu.kis.powp.jobs2d.command;
+
+/**
+ * Interface for objects using visitor.
+ */
+public interface Visitable {

```

Comment:
It should not be a separate interface because Visitor class applies only to commands hierarchy. Please move it to DriverCommand.